src/test/ui/privacy/associated-item-privacy-trait.stderr

```diff
@@ -47,7 +47,7 @@ error: trait `priv_trait::PrivTr` is private
   --> $DIR/associated-item-privacy-trait.rs:25:16
    |
 LL |         let _: <Pub as PrivTr>::AssocTy;
-   |                ^^^^^^^^^^^^^^^^^^^^^^^^
+   |                ^^^^^^^^^^^^^^^^^^^^^^^^ private
```

Comment:
It'd be nice to only emit one of this and the prior error.

Reply:
The hightlight is also not optimal. It should probably be just on the `PrivTr` or at least on the `<Pub as PrivTr>`